### PR TITLE
fix: git error introduced in git 2.35.2

### PR DESCRIPTION
**Pull request by pbhandari356**
Relates to: https://github.com/reviewdog/reviewdog/issues/1158

### CODE REVIEW

**Comment by pbhandari-turo** [comment]
@haya14busa first time contributor, please take a look thanks

**Comment by pbhandari356** [comment]
@haya14busa first time contributor, please take a look thanks!

File: script.sh [hunk 1]
```diff
@@ -1,6 +1,9 @@
 #!/bin/sh
 
-cd "${GITHUB_WORKSPACE}/${INPUT_WORKDIR}" || exit 1
+if [ -n "${GITHUB_WORKSPACE}" ] ; then
+  cd "${GITHUB_WORKSPACE}/${INPUT_WORKDIR}" || exit
```
Comment (shogo82148) [inline script.sh:4]:
```suggestion
  cd "${GITHUB_WORKSPACE}/${INPUT_WORKDIR}" || exit 1
```

**Review by shogo82148 — APPROVED** [review]
LGTM
thanks!

**Comment by review-dog** [comment]
Hi, @pbhandari356! We merged your PR to reviewdog! 🐶
Thank you for your contribution! ✨

**We just invited you to join the @reviewdog organization on GitHub.**
Accept the invite by visiting https://github.com/orgs/reviewdog/invitation.
By joining the team, you'll be a part of reviewdog community and can help the maintenance of reviewdog.

Thanks again!

**Comment by haya14busa** [comment]
I believe the issue does not happen with the composite based action. 

Do we really need to include this fix?
@pbhandari356 @shogo82148

**Comment by shogo82148** [comment]
you are right.
sorry, it's mistake.

**Pull request merged**